### PR TITLE
Add property to disable code coverage for project

### PR DESCRIPTION
**Pull request by pjanotti**
Related to https://github.com/dotnet/corefx/pull/32537#issuecomment-425685002

For now as WIP just to track needed changes.

### CODE REVIEW

**Comment by pjanotti** [comment]
Removing WIP status, this is useful in general to allow finer control on code coverage CI runs (see related issue https://github.com/dotnet/corefx/pull/32537#issuecomment-425685002)

File: src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <!-- Coverage options -->
   <PropertyGroup>
     <CodeCoverageEnabled>false</CodeCoverageEnabled>
-    <CodeCoverageEnabled Condition="'$(SkipTests)' != 'true' and '$(Coverage)' == 'true' and '$(IsPerformanceTestProject)' != 'true'">true</CodeCoverageEnabled>
+    <CodeCoverageEnabled Condition="'$(SkipTests)' != 'true' and '$(Coverage)' == 'true' and '$(IsPerformanceTestProject)' != 'true' and '$(DisableCodeCoverage)' != 'true'">true</CodeCoverageEnabled>
```
Comment (ViktorHofer) [inline src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets]: Why do we need to introduce another property here? Can't we just reuse the existing `CodeCoverageEnabled´ one? i.e.

```
<CodeCoverageEnabled Condition="'$(CodeCoverageEnabled)' == '' and '$(SkipTests)' != 'true' and '$(Coverage)' == 'true' and '$(IsPerformanceTestProject)' != 'true' and '$(DisableCodeCoverage)' != 'true'">true</CodeCoverageEnabled>
<CodeCoverageEnabled Condition="'$(CodeCoverageEnabled)' == ''">false</CodeCoverageEnabled>
```

Comment (weshaggard) [reply src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets]: I tend to agree we should try and use the same property.

Comment (pjanotti) [reply src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets]: Doh! Agreed.

**Pull request merged**